parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -483,7 +499,7 @@ installKubeletAndKubectl() {
   rm -rf ${binPath}/kubelet-* ${binPath}/kubectl-* /home/hyperkube-downloads &
 }
 ensureK8sControlPlane() {
-  if [ -f /var/run/reboot-required ] || [ "$NO_OUTBOUND" = "true" ]; then
```

Comment:
We actually *always* want to start the control plane, even if we're going to reboot at the end of the bootstrap script, because the bootstrap script will depend upon the control plane being online to complete successfully.